### PR TITLE
Update content on Monthly Statistics page

### PR DESCRIPTION
**Pull request by george-bit**
## Context
As part of the Monthly Statistics page we need to update content to explain the data appropriately.

## Changes proposed in this pull request

Update all text and links annotating monthly stats tables

image

## Guidance to review

Please compare review app/locally against https://apply-teacher-training-stats.herokuapp.com/2021-11#applications-by-course-type

I've tried to make things as non cycle dependent as possible to prevent future maintenance. Please let me know if there's anything I've missed.

I am unable to include the file size on the download links however. I've managed to do this before using the `Filesize` gem but for uploaded files. If there's an easier way please let me know.

## Link to Trello card
https://trello.com/c/X7Oy9hU9/4138-use-updated-design-for-the-monthly-report-dashboard)

## Things to check

- [x] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: app/views/monthly_statistics/show.html.erb [hunk 1]
```diff
@@ -85,112 +90,139 @@
...
-    <p class='govuk-body'>For example, if a candidate has received one offer, declined one offer, and is awaiting a provider decision on a third course, they're counted under ‘Received an offer’.</p>
-    <p class='govuk-body'>If a candidate applies again, they are counted only once in the Apply again column.</p>
-    <%= render CandidateInterface::MonthlyStatisticsTableComponent.new(caption: 'Candidates by status (multiple applications counted only once)', statistics: @statistics['candidates_by_status']) %>
-    <%= render CandidateInterface::MonthlyStatisticsTableComponent.new(caption: 'Applications by status', statistics: @statistics['applications_by_status']) %>
+    <p class="govuk-body">These statistics collect data from October 2021 to <%= @monthly_statistics_report.created_at.to_s(:govuk_date) %>
+      (ITT2022) and include deferred applications from the <%= @current_cycle_name %> recruitment cycle (ITT2021).</p>
+    <p class="govuk-body">PNew data for each month in the ITT2022 recruitment cycle is added monthly.</p>
```
Comment (davidgisbey) [inline app/views/monthly_statistics/show.html.erb:113]: `New data for each month in the ITT2022 recruitment cycle is added monthly.`

File: app/views/monthly_statistics/show.html.erb [hunk 2]
```diff
@@ -3,79 +3,84 @@
...
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
-      Applicants for initial teacher training courses: 2021 to 2022
+      Initial teacher training applications data for courses starting in the <%= @academic_year_name %> academic year
     </h1>
 
-    <p class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1">Statistics up to midnight 30 November 2021</p>
-    <p class="govuk-body govuk-!-font-size-16">Next update on 3 January 2022</p>
-
     <div class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1 govuk-!-margin-top-5">Contents</div>
```
Comment (davidgisbey) [inline app/views/monthly_statistics/show.html.erb:9]: ``
govuk-!-font-size-20
``

Comment (frankieroberto) [reply app/views/monthly_statistics/show.html.erb:9]: Actually this should remain `govuk-!-font-size-16` - the prototype is wrong. will fix. 🤦

File: app/views/monthly_statistics/show.html.erb [hunk 3]
```diff
@@ -85,112 +90,139 @@
...
-    </ul>
+    <h2 class="govuk-heading-l" id="data-updates">Data updates</h2>
 
-    <p class='govuk-body'>In the first table below, candidates are only counted once. If they have to more than one course, they’re counted under the highest row (in the table) which applies.</p>
-    <p class='govuk-body'>For example, if a candidate has received one offer, declined one offer, and is awaiting a provider decision on a third course, they're counted under ‘Received an offer’.</p>
-    <p class='govuk-body'>If a candidate applies again, they are counted only once in the Apply again column.</p>
-    <%= render CandidateInterface::MonthlyStatisticsTableComponent.new(caption: 'Candidates by status (multiple applications counted only once)', statistics: @statistics['candidates_by_status']) %>
-    <%= render CandidateInterface::MonthlyStatisticsTableComponent.new(caption: 'Applications by status', statistics: @statistics['applications_by_status']) %>
+    <p class="govuk-body">These statistics collect data from October 2021 to <%= @monthly_statistics_report.created_at.to_s(:govuk_date) %>
```
Comment (davidgisbey) [inline app/views/monthly_statistics/show.html.erb:111]: 👌

File: app/views/monthly_statistics/show.html.erb [hunk 4]
```diff
@@ -3,79 +3,84 @@
...
-    <p class="govuk-body govuk-!-font-size-16">Next update on 3 January 2022</p>
-
     <div class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1 govuk-!-margin-top-5">Contents</div>
 
     <ul class="govuk-list govuk-!-margin-bottom-8 app-toc">
       <li>
         <a href="#introduction" class="govuk-link app-toc__link">
-          <span class="app-toc__number">1.</span>
-          <span class="app-toc__content">Introduction</span>
+          <span class="app-toc__content">What's included in this data?</span>
```
Comment (frankieroberto) [inline app/views/monthly_statistics/show.html.erb:14]: Can we keep all the numbers within the links please - and they should all have a number, including the first section ("What's included in this data?").  Prototype is wrong, will fix.

File: app/views/monthly_statistics/show.html.erb [hunk 5]
```diff
@@ -85,112 +90,139 @@
...
+      <li>application rejected by provider (least “favourable”)</li>
+    </ul>
+
+    <p class='govuk-body'>Candidates can make one application at a time, as many times as they like, if their first set
+      of applications do not lead to a place.</p>
+    <p class='govuk-body'>The ‘apply again’ column of the first table below counts only the status of a candidate's most
+       recent ‘apply again’ application. Their previous applications are not counted anywhere in the table.</p>
+    <p class='govuk-body'>The status of any one application changes throughout its lifecycle. Statuses are captured at
+      the end of the data period.</p>
+    <%= render CandidateInterface::MonthlyStatisticsTableComponent.new(caption: 'Candidates by most “favourable” application status', statistics: @statistics['candidates_by_status']) %>
```
Comment (davidgisbey) [inline app/views/monthly_statistics/show.html.erb:150]: `This table counts all applications by their status.` is missing between the tables

**Comment by davidgisbey** [comment]
LGTM other than those nits. Will let Frankie sign it off 👍

**Review by davidgisbey — APPROVED** [review]

**Review by frankieroberto — APPROVED** [review]

**Review by frankieroberto — CHANGES_REQUESTED** [review]
I've got some last-minute changes, but I'll push some commits direct to the branch.

**Review by frankieroberto — APPROVED** [review]

**Pull request merged**